x-pack/elastic-agent/pkg/agent/cmd/enroll_cmd.go

```diff
@@ -184,8 +185,16 @@ func newEnrollCmdWithStore(
 func (c *enrollCmd) Execute(ctx context.Context, streams *cli.IOStreams) error {
 	var err error
 	defer c.stopAgent() // ensure its stopped no matter what
+	span, ctx := apm.StartSpan(ctx, "enroll", "app.internal")
+	defer func() {
+		if err != nil {
+			apm.CaptureError(ctx, err).Send()
+		}
```

Comment:
[Suggestion]
It's safe to call `apm.CaptureError` with a nil error and then call `Send`. Therefore it can be simplified.
```suggestion
		apm.CaptureError(ctx, err).Send()
```